lib/map.jsx

```diff
@@ -0,0 +1,28 @@
+var React = require('react');
+
+var MapComponent = React.createClass({
+
+    propTypes: {
+      accessToken: React.PropTypes.string.isRequired,
+      mapId: React.PropTypes.string.isRequired,
```

Comment:
(Actually, if we end up getting rid of the `id` attribute, I think it's fine to keep this name.)